src/core/handlers/GraphQLHandler.test.ts

```diff
@@ -160,6 +160,12 @@ describe('parse', () => {
       })
 
       expect(await handler.parse({ request })).toEqual({
```

Comment:
We also need to add the URL matching unit tests to the GraphQL handler:

- parses a graphql request if the given endpoint matches (for graphql.link)
- skips parsing if the given endpoint does not match (for graphql.link)

I assume the endpoint-less requests should be covered by the existing tests but an extra look at those won't hurt.

Reply:
added cases for this here: https://github.com/mswjs/msw/pull/1871/files#diff-81d8e35b978b65087de9c9c140e9cc932591997c3d5a464106b07d6fb0094fe5R374-R527

we could add more, but it seems like they'd end up being relatively duplicative of other cases already covered. this may already bit a bit more than necessary

Reply:
Thanks! My thinking as well, no need to duplicate what's already covered by existing unit tests.